hyperjumptech/monika-config-generator
# sendgrid: add sender field

This PR resolves #85 
### How to test
- `npm install`
- `npm run dev`
- Visit `localhost:3000`
- Create config with guided mode (menu **I'm new to Monika**)
- Create config with advanced mode (menu **I have used Monika before**)
- Both set notification channel to SendGrid
- Download config and verify result
